Jenkinsfile

```diff
@@ -84,14 +83,6 @@ pipeline {
             }
           }
         }
-
-        stage('Build nrf samples') {
```

Comment:
This was removed because it's now done by CI in `fw-nrfconnect-nrf`, correct?

Reply:
Correct